helpers/publicarMovimientoDetalle.go

```diff
@@ -0,0 +1,190 @@
+package helpers
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/astaxie/beego/logs"
+	"github.com/udistrital/movimientos_crud/models"
+	"github.com/udistrital/utils_oas/errorctrl"
+)
+
+func PublicarMovimientosDetalle(idMovProcExterno int) (movimientosDetalleRespuesta []models.MovimientoDetalle, outputError map[string]interface{}) {
```

Comment:
Falta documentar, cambiar el `...` del _suggested change_
```suggestion
// PublicarMovimientosDetalle ...
func PublicarMovimientosDetalle(idMovProcExterno int) (movimientosDetalleRespuesta []models.MovimientoDetalle, outputError map[string]interface{}) {
```